cdap-common/bin/functions.sh

```diff
@@ -207,13 +207,7 @@ cdap_stop_pidfile() {
 #
 cdap_check_pidfile() {
   local readonly __pidfile=${1} __label=${2:-Process}
-  if [[ -f ${__pidfile} ]]; then
-    local readonly __pid=$(<${__pidfile})
-    if [[ $(cdap_status_pidfile ${__pidfile} ${__label} >/dev/null) ]]; then
-      echo "${__label} running as PID ${__pid}. Stop it first."
```

Comment:
In the case of running start while already running, the new output will be ``${__label} running as PID ${__pid}`` which can easily be misconstrued as a successful command.

Maybe add equivalent output where this is called, ie https://github.com/caskdata/cdap/blob/a4d03ef1922cb6ac145bd0f670a85d3a0c2e12ea/cdap-common/bin/functions.sh#L612